local_libs/lib-dex-commons/src/main/kotlin/gov/cdc/dex/mmg/MmgUtil.kt

```diff
@@ -63,7 +63,7 @@ class MmgUtil(val redisProxy: RedisProxy)  {
     @Throws(InvalidConditionException::class)
     // Populates DexMessageInfo, including list of MMGs and provision route.
     fun getMMGMessageInfo(msh21_2: String, msh21_3: String?, eventCode: String, jurisdictionCode: String?): DexMessageInfo {
-        val messageInfo = DexMessageInfo(eventCode, null, null, jurisdictionCode)
+        val messageInfo = DexMessageInfo(eventCode, null, null, jurisdictionCode, null)
```

Comment:
Make this default to "ECR" for backwards compatibility ( I believe only case messages will call this method, so the default to ECR should work ok here.

Reply:
I have updated the method to have type as "ECR"